rllib/agents/trainer.py

```diff
@@ -503,7 +503,12 @@
         "policy_map_cache": None,
         # Function mapping agent ids to policy ids.
         "policy_mapping_fn": None,
-        # Optional list of policies to train, or None for all policies.
+        # Specifies those policies that should be updated.
+        # Options are:
+        # - None for all policies.
+        # - An iterable of PolicyIDs to be updated.
```

Comment:
nit nit nit
maybe just "An iterable of PolicyIDs"
I am not sure if "to be updated" applies to the list of ids, or the policies specified via these ids. when reading this.

Reply:
Fixed and clarified.